app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -166,6 +176,40 @@ public Study getStudy(StudyIdentifier studyId) {
         return studyDao.getStudies();
     }
 
+    public Study createStudyAndUser(StudyAndUserHolder studyAndUserHolder) throws SynapseException {
+        checkNotNull(studyAndUserHolder, Validate.CANNOT_BE_NULL, "study and users");
+        if (studyAndUserHolder.getUsers() == null) {
+            throw new BadRequestException("User list cannot be null.");
+        }
+        if (studyAndUserHolder.getStudy() == null) {
+            throw new BadRequestException("Study cannot be null.");
+        }
+
+        List<StudyParticipant> users = studyAndUserHolder.getUsers();
+        if (users.isEmpty()) {
+            throw new BadRequestException("User list should not be empty.");
+        }
+
+        Study study = studyAndUserHolder.getStudy();
+
+        // first create study
+        study = createStudy(study);
+
+        // then create users for that study
+        // send verification email as well
+        for (StudyParticipant user: users) {
+            IdentifierHolder identifierHolder = participantService.createParticipant(study, user.getRoles(), user,true);
```

Comment:
We should check that the roles only contain Developer or Researcher or both. We don't want people to be able to create more admins or worker accounts with this API. Also, a user with no roles is probably not appropriate here.